src/main/java/io/arivera/oss/embedded/rabbitmq/OfficialArtifactRepository.java

```diff
@@ -15,10 +15,17 @@
  */
 public enum OfficialArtifactRepository implements ArtifactRepository {
 
-  RABBITMQ("http://www.rabbitmq.com/releases/rabbitmq-server/v%s.%s.%s/rabbitmq-server-%s-%s.%s"),
-  GITHUB("https://github.com/rabbitmq/rabbitmq-server/releases/download/rabbitmq_v%s_%s_%s/rabbitmq-server-%s-%s.%s"),
+  GITHUB("https://github.com/rabbitmq/rabbitmq-server/releases/download/%sv%s/rabbitmq-server-%s-%s.%s", VersionSupport.ANY),
```

Comment:
Oh, one more recommendation: keeping the enum order the same as before, and only adding new ones at the end, help ensure binary backwards compatibility in some cases.